pkg/registry/capability/capability.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//use for --watch-cache-sizes param of kube-apiserver
+//make watch cache size of resources configurable
+package capability
```

Comment:
I agree that capability is confusing.

I suggest: cachesize or watchcachesize
Although the former is probably better, since we are using the cache also to serve lists now, not only watch...